src/ClusterBootstrap/services/logging/kibana.yaml

```diff
@@ -1,43 +1,42 @@
-kind: DaemonSet
 apiVersion: apps/v1
+kind: DaemonSet
 metadata:
-  name: kibana-logging
+  name: kibana
   namespace: kube-system
   labels:
-    k8s-app: kibana-logging
+    k8s-app: kibana
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
   selector:
     matchLabels:
-      k8s-app: kibana-logging
+      k8s-app: kibana
   template:
     metadata:
       labels:
-        k8s-app: kibana-logging
+        k8s-app: kibana
     spec:
       hostNetwork: true
       nodeSelector:
         kibana: active
       containers:
-      - name: kibana-logging
-        image: docker.elastic.co/kibana/kibana:5.6.2
+      - name: kibana
+        image: docker.elastic.co/kibana/kibana:6.8.5
```

Comment:
please change the docker image to configurable parameter. Default value can be found in params.py
Also, please push official docker image to our dlws repo and use the images in dlws repo to avoid dependence -- this applies to all the docker images used in this PR.

Reply:
Done need a credential to push all these images to dlws namespace of docker.io